tests/e2e/test-worker/index.ts

```diff
@@ -381,7 +389,13 @@ console.log('Terminal server on port ' + port);
 
       // Process start
       if (url.pathname === '/api/process/start' && request.method === 'POST') {
-        const process = await executor.startProcess(body.command);
+        const process = await executor.startProcess(body.command, {
+          processId: body.processId,
+          env: body.env,
+          cwd: body.cwd,
+          timeout: body.timeout,
+          autoCleanup: body.autoCleanup
```

Comment:
Are the rest of these fields actually being used properly? I would say we should not add them to the scope of this PR, but if not addressed, we should create a separate issue to track. #81 was created a while ago for the timeout.